### PR TITLE
feat: add v3 endpoint for enterprise learner summary endpoint

### PR DESCRIPTION
**Pull request by muhammad-ammar**
**Description:** Add api gateway spec for version v3 of `learner-summary` endpoint so that LPR V1 API can be accessed through new learner-summary endpoint.

**JIRA:** https://openedx.atlassian.net/browse/ENT-4881

**Dependency:** https://github.com/edx/edx-enterprise-data/pull/264

### CODE REVIEW

**Review by saleem-latif — APPROVED** [review]
Looking Good so far

File: swagger/api.yaml [hunk 1]
```diff
@@ -68,6 +68,9 @@ paths:
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise/260055b/api.yaml#/endpoints/v1/enterpriseCustomerCourseEnrollments"
   "/enterprise/v2/enterprise-customer/{uuid}/learner-summary":
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/8fc95e4/api.yaml#/endpoints/v1/enterpriseCustomerLearnerSummary"
+  # Learner Summary endpoint powered by Learner Progress Report V1 API
+  "/enterprise/v3/enterprise-customer/{uuid}/learner-summary":
+    $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/*******/api.yaml#/endpoints/v2/enterpriseCustomerLearnerSummary"
```
Comment (muhammad-ammar) [inline swagger/api.yaml:73]: **Note:** commit hash will be updated once https://github.com/edx/edx-enterprise-data/pull/264 is merged

File: swagger/api.yaml [hunk 2]
```diff
@@ -68,6 +68,9 @@ paths:
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise/260055b/api.yaml#/endpoints/v1/enterpriseCustomerCourseEnrollments"
   "/enterprise/v2/enterprise-customer/{uuid}/learner-summary":
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/8fc95e4/api.yaml#/endpoints/v1/enterpriseCustomerLearnerSummary"
```
Comment (muhammad-ammar) [inline swagger/api.yaml:70]: **Question to reviewers:** Do I also need to update the commit hash for other `edx-enterprise-data` endpoints?

Comment (georgebabey) [reply swagger/api.yaml:70]: Good question - I don't believe its required but I would anyways just to keep them all consistent

**Review by georgebabey — APPROVED** [review]

**Comment by mraarif** [comment]
#128 contains a fix for `libcurl4-openssl-dev` installation issue

**Pull request merged**